orc8r/cloud/easy_deploy/docker/docker_root/scripts/playbooks/roles/services/tasks/addcerts.yml

```diff
@@ -0,0 +1,38 @@
+---
+- name: check if required infra variables are set
+  ansible.builtin.script: "{{ scripts_dir }}/deploy_cli.py configure --infra --check"
+  register: result
+  failed_when: result.rc != 0
+
+- name: Open configuration file
+  shell: "cat {{ configs_dir }}/infra.tfvars.json"
+  register: result
+
+- name: Set Infra Configs
+  set_fact:
+    infra_configs: "{{ result.stdout | from_json }}"
+
+- name: create certs directory
+  file:
+    name: "{{ secrets_dir }}"
+    state: directory
+    mode: '0775'
+
+- name: generate self-signed certs
```

Comment:
Support for trusted (non-self-signed) certs?

Reply:
For now, I will leave it as is and add support for non self signed certs later. User can always manually add trusted certs in the deployment secrets directory

Reply:
Added an option to skip generating self signed and only generate domain certs based on trusted certs